addons/html_editor/static/src/editor/core/delete_plugin.js

```diff
@@ -78,29 +62,93 @@ export class DeletePlugin extends Plugin {
     // commands
     // --------------------------------------------------------------------------
 
-    deleteRange() {
-        let selection = this.shared.getEditableSelection();
+    deleteSelection() {
```

Comment:
maybe deleteSelection could take the selection as argument? it is only called at two locations, which already have the selection?